### PR TITLE
[Atlantis] Adding possibility to specify a storageClass for the PVC

### PR DESCRIPTION
**Pull request by vlandemaine-orange**
Signed-off-by: Valentin Landemaine <valentin.landemaine.ext@orange.com>

# Helm Atlantis

#### Enable the possibility to specify a storageClass for the PVC mounted on the container

@jkodroff
@lkysow 
@jeff-knurek 
@sstarcher 

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by jeff-knurek** [comment]
the CI fails, you'll need to bump the chart version, but otherwise lgtm

**Comment by vlandemaine-orange** [comment]
@jeff-knurek Excuse me, but what do you mean by "bump the chart version" ? I've made the fork one hour ago.

**Comment by jeff-knurek** [comment]
> ci/circleci: lint-charts — Your tests failed on CircleCI
[Details](https://circleci.com/gh/helm/charts/62207?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Hmm, I was actually surprised it wasn't spelled out here: https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
but instead just says that the linter must pass

**Comment by vlandemaine-orange** [comment]
I didn't understand why the linter not pass.. 

`Checking chart 'stable/atlantis' for a version bump...
Old chart version: 1.1.6
New chart version: 1.1.6
Error linting charts`

**Comment by jeff-knurek** [comment]
ah, basically it's say that you made a change, but didn't change the version 

so, for me as a user, if I try to deploy chart `v1.1.6`, will I get it with your change or without - the linter is preventing that from happening by telling you to change the version ;)

**Comment by vlandemaine-orange** [comment]
Oh okay ! Now it's good :)

**Comment by sstarcher** [comment]
/lgtm

@vlandemaine-orange you will need to pass the DCO check.

File: stable/atlantis/README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
...
 | `allow_repo_config` | Whether to allow the use of [atlantis.yaml files](https://www.runatlantis.io/docs/atlantis-yaml-reference.html). | `false`                  |
 | `atlantis_data_storage`                     | The amount of storage available for Atlantis' data directory (mostly used to check out git repositories).                                                                                                                                                                                                                                                  | `5Gi`                  |
+| `atlantis_data_storageClass`                     | Storage class of the volume mounted for the Atlantis data directory.                                                                                                                                                                                                                                                  | n/a                  |
```
Comment (lkysow) [inline stable/atlantis/README.md:35]: shouldn't these all be camelCase?

Comment (lkysow) [reply stable/atlantis/README.md:35]: Also why is it prefixed with `atlantis_`? I don't think that's necessary given this is the Atlantis chart.

**Pull request merged**